### PR TITLE
fix(9anime): fix details

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Small temporary fix for details on some sites [e.g.](https://9anime.vc/). (I have put 9anime on my to-do list for a re-write. but this PR fixes that it works untill then.)
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/0-9/9anime/presence.ts [hunk 1]
```diff
@@ -30,53 +30,56 @@ presence.on("UpdateData", async () => {
 		!isNaN(video.duration) &&
 		document.location.pathname.includes("/watch/")
 	) {
-		const [startTimestamp, endTimestamp] = presence.getTimestamps(
-				Math.floor(video.currentTime),
-				Math.floor(video.duration)
-			),
-			coverArt = document.querySelector<HTMLImageElement>("#w-info img")?.src,
+		delete presenceData.startTimestamp;
+		[, presenceData.endTimestamp] = presence.getTimestamps(
```
Comment (Slowlife01) [inline websites/0-9/9anime/presence.ts:34]: Don't see why we should remove startTimestamp

Comment (DarkVillager) [reply websites/0-9/9anime/presence.ts:34]: > Don't see why we should remove startTimestamp
Ok

Comment (EncryptedDev) [reply websites/0-9/9anime/presence.ts:34]: Is this suggestion going to be taken on board or forgotten about?

**Comment by DarkVillager** [comment]
Currently working on the rewrite anyway.

**Pull request closed without merging**